examples/auth.py

```diff
@@ -16,5 +16,31 @@ async def run():
         logging.info(await obo_auth_session.session_token)
 
 
-logging.basicConfig(level=logging.DEBUG)
+logging.config.dictConfig({
+    "version": 1,
+    "disable_existing_loggers": True,
+    "filters": {"taskfilter": {"()": "symphony.bdk.core.client.api_client_factory.TraceIdFilter"}},
```

Comment:
we have to register it here, can we do it programatically to avoid this to the user?